Client/Frontend/Browser/Playlist/Playlist.xcdatamodeld/Playlist.xcdatamodel/contents

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
This can be deleted as it looks like Playlist data is in regular CoreData store now